apps/modernization-ui/src/apps/patient/data/identification/asIdentification.ts

```diff
@@ -6,7 +6,7 @@ const asIdentification = (entry: IdentificationEntry): Identification => {
     const { type, issuer, ...remaining } = entry;
 
     return {
-        type: asValue(type),
+        type: asValue(type) ?? '',
```

Comment:
suggestion:  I encountered a similar thing with [asEthnicity](https://github.com/CDCgov/NEDSS-Modernization/blob/ca26e4ce518d92a3fca50bf26d7546675f13d75b/apps/modernization-ui/src/apps/patient/data/ethnicity/asEthnicity.ts#L8) and decided to make it return `undefined` if the required values were not present.